src/main/java/Bonnie/Task.java

```diff
@@ -0,0 +1,32 @@
+package Bonnie;
+
+abstract public class Task {
```

Comment:
```suggestion
public abstract class Task {
```

will be flagged out by CheckStyle if you use it. Out of JLS suggestion order.